src/pages/stations/detail/components/StationDetails.tsx

```diff
@@ -281,30 +282,30 @@ const StationDetails = ({ stationDetail, formatedDate, goBack }: Prop) => {
                 </Grid>
                 <Grid item xs={9}>
                   <Typography variant="body2" fontWeight={'fontWeightMedium'}>
-                    {stationDetail?.targetHostPof}
+                    {stationDetail?.targetHostPof ? stationDetail.targetHostPof : '-'}
                   </Typography>
                 </Grid>
                 <Grid item xs={3}>
                   <Typography variant="body2">{t('stationDetailPage.service')}</Typography>
                 </Grid>
                 <Grid item xs={9}>
                   <Typography variant="body2" fontWeight={'fontWeightMedium'}>
-                    {stationDetail?.targetPathPof}
+                    {stationDetail?.targetPathPof ? stationDetail.targetPathPof : '-'}
                   </Typography>
                 </Grid>
                 <Grid item xs={3}>
                   <Typography variant="body2">{t('stationDetailPage.port')}</Typography>
                 </Grid>
                 <Grid item xs={9}>
                   <Typography variant="body2" fontWeight={'fontWeightMedium'}>
-                    {stationDetail?.targetPortPof}
+                    {stationDetail?.targetPortPof ? stationDetail.targetPortPof : '-'}
                   </Typography>
                 </Grid>
                 <Grid item xs={12} mt={2} sx={{ display: 'flex', justifyContent: 'space-between' }}>
                   <Typography variant="sidenav">{t('stationDetailPage.associatesEC')}</Typography>
                   <ButtonNaked
                     component="button"
-                    onClick={() => history.push(ROUTES.STATION_EC_LIST, { stationId })}
+                    onClick={() => history.push(`${BASE_ROUTE}/stations/${stationId}/ec-list`)}
```

Comment:
Use the 
`<ButtonNaked
                    component={Link}
                    to={generatePath(ROUTES.ROUTES.STATION_EC_LIST, {
                      stationId: stationDetail.stationId,
                    })}`

codebase convention so we can manage and maintain routes more easely